slices/redis-server.yaml

```diff
@@ -0,0 +1,29 @@
+package: redis-server
+
+essential:
+  - redis-server_copyright
+
+slices:
+  bins:
+    essential:
+      - redis-server_config
+      - redis-tools_bins
+    contents:
+      /usr/bin/redis-server:
+
+  config:
+    contents:
+      /etc/default/redis-server:
+      /etc/logrotate.d/redis-server:
```

Comment:
Also, should I remove this logrotate config, since it's only useful if you have logrotate installed?
Since no slices exist for logrotate, it is essentially a waste of space?

Reply:
Yes that's a good idea. We may later re-introduce it if necessary, as a dedicated `logrotate` slice for example

Reply:
Done.